Gum/Managers/NameVerifier.cs

```diff
@@ -244,25 +244,25 @@ private void IsNameAlreadyUsed(string name, object objectToIgnore, ElementSave e
 
             if (objectToIgnore != elementSave && name == elementSave.Name)
             {
-                whyNotValid = "The element is named " + elementSave.Name;
+                whyNotValid = $"The element is named '{elementSave.Name}'";
```

Comment:
This (and the rest of the checks) was also modified because the message was sometimes a bit confusing, but maybe there is a good reason why it was like this, so I'd leave that to your judgement.

Reply:
This validation check should read something to the effect of:

"The element containing this is named {elementSave.Name}, so this cannot be named the same name."

In C#, a member in a class cannot be named the same as the class itself. Since Gum is often used to codegen, naming something within an Element the same name as the element itself will cause compile errors.  Gum could ignore this, but we enforce this rule in Gum for the sake of errors downstream.

Please feel free to change this error message to be clearer, using the phrasing above if you feel it makes sense.

Reply:
Yep, maybe I should have commented on the next lines, as this line didn't change much, but after this one, I've replaced `instance.ToString()` with `instance.Name` for example.

Not sure if 1. that can cause any issues and 2. is something you want, mainly because now instead of "Container Thing of Test" you'll only get "Thing" (because at least for the errors I got, having the whole phrase was a bit strange).

Reply:
If that reads better, that's fine by me.